chore(docs) Remove instances of let's to resolve mdx lint warnings (#47013)
supabase/supabase: apps/docs/content/guides/ai/rag-with-permissions.mdx

);
```

> Notice how we record the `owner_id` on each document. Create an RLS policy that restricts access to `document_sections` based on whether or not they own the linked document:

Author: ```suggestion
Notice the record of `owner_id` on each document. Create an RLS policy that restricts access to `document_sections` based on whether or not they own the linked document:
```
